src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -38,17 +38,23 @@
  */
 class PropertyAccessor implements PropertyAccessorInterface
 {
+    /** @var int Allow none of the magic methods */
+    public const DISALLOW_MAGIC_METHODS = ReflectionExtractor::DISALLOW_MAGIC_METHODS;
+    /** @var int Allow magic __get methods */
+    public const MAGIC_GET = ReflectionExtractor::ALLOW_MAGIC_GET;
```

Comment:
These are now aliases to the `ReflectionExtractor` constants, since it would be unexpected/not-discoverable for the user to use `ReflectionExtractor` consts to configure the accessor.